src/CoreBundle/Event/FilterResponseEvent.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+final class FilterResponseEvent extends Event
```

Comment:
There is an event with the same name in Symfony - we shouldn't "overwrite" (name collide) this.

Add "content" to the name?

Reply:
The fact, that the original event is called like that was the reason I named this one like I did in the first place. They are in different namespaces so they do have different names technically. But I see your point. If we want to differentiate both, I'd suggest FilterChameleonResonposeEvent.. But I am not sure about that either.